go.mod

```diff
@@ -0,0 +1,17 @@
+module go.viam.com/slam
+
+go 1.19
+
+require (
+	github.com/edaniels/golog v0.0.0-20230215213219-28954395e8d0
+	github.com/pkg/errors v0.9.1
+	go.viam.com/test v1.1.0
```

Comment:
Is there a reason why we are using a different version than rdk? https://github.com/viamrobotics/rdk/blob/main/go.mod#L86

Reply:
could have it match. it probably doesn't matter. I'm assuming all things here took latest.